spec/features/record_deployment_spec.rb

```diff
@@ -0,0 +1,28 @@
+#
+# pact-broker record-deployment --pacticipant Foo --version 1 --environment test --end-previous-deployment
+#
+
```

Comment:
@mefellows thoughts?

Reply:
I can't think of a better term for `--end-previous-deployment`, but docs should be able to easily address this (replace comes to mind, but it has it's own set of problems).

As per above, would it be worth having the option at least of recording the status of the deployment (pass/fail), and other data such as duration, deployment notes, link to system etc.?

Reply:
Maybe... we can always add it later, but I'm not feeling like it's part of our core concern at the moment.